packages/jitar-reflection/src/parser/Parser.ts

```diff
@@ -227,10 +227,10 @@ export default class Parser
         {
             case Keyword.IMPORT:
                 return this.#parseImport(tokenList);
-            
+
```

Comment:
This may cause a merge conflict with #166. Can this be reverted?

Reply:
We've discussed this and leave it like this. We'll see what happens when at merge.